### ISSUE TITLE
macOS build failure against 10.8 SDK

### ISSUE THREAD

**Issue by sezero** [libsdl-org/SDL#4957]
```
src/video/cocoa/SDL_cocoawindow.m:423:39: warning: instance method '-contentLayoutRect' not found (return type defaults to 'id') [-Wobjc-method-access]
            rect.origin.y = [nswindow contentLayoutRect].size.height - mouse_rect.y - mouse_rect.h;
                                      ^~~~~~~~~~~~~~~~~
/opt/MacOSX10.8.sdk/System/Library/Frameworks/AppKit.framework/Headers/NSWindow.h:188:12: note: receiver is instance of class declared here
@interface NSWindow : NSResponder <NSAnimatablePropertyContainer, NSUserInterfaceValidations, NSUserInterfaceItemIdentification>
           ^
src/video/cocoa/SDL_cocoawindow.m:423:58: error: property 'size' not found on object of type 'id'
            rect.origin.y = [nswindow contentLayoutRect].size.height - mouse_rect.y - mouse_rect.h;
                                                         ^
1 warning and 1 error generated.
```

AFAIU, `contentLayoutRect` is a 10.10 thing.

**Comment by slouken** [comment]
Can you verify this is fixed by https://github.com/libsdl-org/SDL/commit/cc094f4d7c6c0c338f3add9167e7d22373d43c96?

**Comment by sezero** [comment]
The compile part is fixed, linkage still fails similar to bug #4958 
Closing: The bug to be fixed now is #4958.
